com.microsoft.java.debug.core/src/main/java/com/microsoft/java/debug/core/adapter/DebugAdapterContext.java

```diff
@@ -38,6 +38,7 @@
     private Charset debuggeeEncoding;
     private transient boolean vmTerminated;
     private boolean isVmStopOnEntry = false;
+    private boolean isDebugging = true;
```

Comment:
I don't see you change the `LaunchRequestHandler` logic, please remove the debug options from launching command, too.